### ISSUE TITLE
document `Error: account sequence mismatch` during `agoric deploy`

### ISSUE THREAD

**Issue by dckc** [Agoric/documentation#458]
I'm doing `agoric deploy api/deploy.js` in a dapp to the devnet. It's taking a while, so I check wallet logs and I see:

```
ag-solo_1  | Error#6: Command failed: ag-cosmos-helper tx swingset deliver --keyring-backend=test @/tmp/ag-solo-cosmos-deliver.2021225-1-xgxf5q.2jbco --gas=auto --gas-adjustment=1.2 --from=ag-solo --broadcast-mode=block --yes --chain-id=agoricdev-1 --node=tcp://157.245.194.200:26657 --home=/usr/src/agoric-sdk/packages/cosmic-swingset/solo/agoric/ag-cosmos-helper-statedir
ag-solo_1  | Error: account sequence mismatch, expected 39, got 38: incorrect account sequence: invalid request
```

It looks scary, but it seems to be benign; progress seems to resume within a minute or so.

@michaelfig what's going on here?

When we have an answer, let's put it near the docs for `agoric deploy`, please.

**Comment by Tyrosine22** [comment]
As far as I know, we don't currently document any error or log messages. Thus,, this task should probably be closed & replaced with a more general purpose "document all error & log messages" task.

**Comment by dckc** [comment]
Resist the urge to generalize!

That is:

Please don't let fixing *all* the world's problems that people might theoretically encounter get in the way of fixing *one* of the world's problems that I actually encountered.

And there is some precedent for troubleshooting docs, at least in the validator guide (iou link)

**Comment by Tyrosine22** [comment]
Yeah, this could be framed as part of a troubleshooting section. I'd prefer to be thorough & doc all the error messages and/or log outputs. Either way, I won't get to this for quite some time, so... **shrug**.
